src/components/organisms/CertificationPanel.tsx

```diff
@@ -0,0 +1,47 @@
+import { useRouter } from "next/router";
+import { AiFillEye } from "react-icons/ai";
+import { config } from "../../env";
+import { Certification } from "../../utils/types";
+
+const CertificationPanel = ({
+  certification,
+}: {
+  certification: Partial<Certification>;
+}) => {
+  const { name } = certification;
+  const { push } = useRouter();
+
+  const linkToCertification = new URL(
+    `${config.nextPublicBaseUrl}/certifications/${encodeURIComponent(name)}`
+  );
+
+  return (
+    <>
+      <div
+        className={`flex flex-row bg-light-light dark:bg-dark-light px-4 py-4 mx-2 my-1 rounded-lg items-center cursor-pointer border border-light-light dark:border-dark-light hover:bg-light-dark hover:border-light-graybutton hover:dark:bg-dark-radargrid hover:dark:border-dark-graybutton`}
+      >
+        <div className={`flex flex-col w-full`}>
+          <div className="flex flex-row justify-between">
+            <div className="flex flex-col">
+              <h2 className="text-xl cursor-pointer">{name}</h2>
+            </div>
+          </div>
+        </div>
+        <div
+          className="flex w-1/6 justify-end cursor-pointer"
+          onClick={(e) => {
+            e.stopPropagation();
+            push(linkToCertification);
+          }}
+        >
+          <AiFillEye
```

Comment:
I think we should use same way than the skill panels : 

![Capture d’écran de 2023-02-15 10-39-16](https://user-images.githubusercontent.com/85115963/218990963-8fb54986-89a6-4cc7-8db4-84d5e43acfd9.png)

With the logo and the number of user. And by the way, I would suggest to click on this button, for the skill panel as well, to access to the user panel. And so, delete the eye. What do you think about that ?

Reply:
Yes, you're right! I've just committed the changes